app/deck/1.8.x/guides/environment-variables.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Using environment variables with decK
+toc: false
+---
+
+When you use deck to apply configurations to Kong, decK reads data in plain text on a state file by default. To add more security, you can also store sensitive information, for example `apiKey`, `client_secret` in environment variables. decK can then read data directly from environment variables and apply it.
+
+What you need to do is to create environment variables with `DECK_` prefix and reference them as `{%raw%}${{ env "DECK_*" }}{%endraw%}` on your state file.
+
+Below example demonstrates how to apply apiKey stored in environment variable. It enables [key authentication][key-auth] plugin globally, create a consumer `demo` with an apiKey.
```

Comment:
```suggestion
The following example demonstrates how to apply an apiKey stored in an environment variable. 
You can use this method for any sensitive content. 
```
I'm moving the info about the global plugin and consumer into the steps where that happens.